### DISCUSSION TITLE
Add label on a device_bay.yml

### DISCUSSION THREAD

**Discussion by NTNTP** [netbox-community/ansible_modules#862] [General]
Hello Everyone,

I'm currently using ansible an netbox to provide un inventory in my company. 

I'm using the netbox_device_bay.yml to add devices in Netbox but when i try to enter the label module, it said to me that the module: data.label. is not supported. 

And i'was wondering if the label module is really not exiting in the file or if it is somwhere else ? 

If you can help me to find if it is available or not ?

**Comment by rodvand** [comment]
If you look at https://netbox-ansible-collection.readthedocs.io/en/latest/plugins/netbox_device_bay_module.html#ansible-collections-netbox-netbox-netbox-device-bay-module you will find the fields currently supported. Label is not there at the moment, so it will have to be added. It's a fairly simple task, so if you open a FR on this it will likely be added for the next collection release.

**Reply by NTNTP** [reply]
Thank you for the quick response i look forward to get the new version of the collection. 

Have a nice day

**Comment by sc68cal** [comment]
Created an issue to cover the enhancement, based on this discussion

**Reply by NTNTP** [reply]
I see on the Ansible website, that the label module will be added on the 3.9.0 version, do i still need to create an issue about it ?

**Reply by sc68cal** [reply]
No, the issue was https://github.com/netbox-community/ansible_modules/issues/864 and it has been completed
